helix-term/src/ui/mod.rs

```diff
@@ -506,18 +506,31 @@ pub mod completers {
             }) // TODO: unwrap or skip
             .filter(|path| !path.is_empty());
 
+        // TODO: use a custom theme key e.g. "ui.text.directory"
+        let directory_color = editor.theme.get("function");
+
+        let style_from_file = |file: Cow<'_, str>| {
+            if file.ends_with('/') {
```

Comment:
This should check for `std::path::MAIN_SEPARATOR` so it works on Windows as well

Reply:
Ok, I did it